invoice2data/extract/loader.py

```diff
@@ -35,6 +39,29 @@ def read_templates(folder=None):
     Load yaml templates from template folder. Return list of dicts.
 
     Use built-in templates if no folder is set.
+
+    Parameters
+    ----------
+    folder : str
+        user defined folder where they stores their files, if None uses built-in templates
+
+    Returns
+    -------
+    output : Instance of `InvoiceTemplate`
+        template which match based on keywords
+
+    Examples
+    --------
+
+    >>> read_template("~/home/duskybomb/invoice-templates/")
```

Comment:
This won't work because Python doesn't expand the `~` char. It also doesn't make sense because you already included your home folder.

Reply:
Since just reading the some templates isn't useful, it would be good to show what to do with them later.